src/pages/uxp/reference-js/Modules/FileSystem.md

```diff
@@ -147,13 +147,13 @@ const fd = await fs.open("plugin-data:/fileToRead.txt", "r");
 ## close(fd, callback)
 Closes a file descriptor asynchronously
 
+**Returns**: `int` - 0 if succeeded, otherwise throws an Error  
+
```

Comment:
https://github.com/AdobeDocs/uxp/issues/86

Reply:
Is the return type just number or Promise<number> ?